docs/tutorials/integrated/soil_canopy_tutorial.jl

```diff
@@ -306,7 +299,7 @@ canopy_model_args = (; parameters = shared_params, domain = canopy_domain);
 # atmospheric and radiative flux conditions from the observations at the Ozark
 # site as was done in the previous tutorial.
 
-land_input = (atmos = atmos, radiation = radiation)
+land_input = (atmos = atmos, radiation = radiation, soil_organic_carbon = Csom)
```

Comment:
we pass this in, now, and the soil co2 driver is created in the LandModel constructor